### ISSUE TITLE
NTR: 'negative regulation of interleukin-37 biosynthetic process' 

### ISSUE THREAD

**Issue by ShirinSaverimuttu** [geneontology/go-ontology#18370]
Request for 'negative regulation of interleukin-37 biosynthetic process' 

Evidence - PMID: 30362558. Figure 2D:

> "The expression of IL-37 (mRNA) was reduced in miR-657 overexpressed THP-1 cells, which were induced by PMA".

Currently no IL-37 production terms exist.

cc
@BarbaraCzub 
@RLovering

**Comment by ukemi** [comment]
I know we have been adding a lot of these, but do we really want to go down this route using pre-composed terms?

**Comment by pgaudet** [comment]
I have said before - I dont think those terms 'production of x' (and 'secretion of x') tell us much about the biology. It's just an observation.

**Comment by addiehl** [comment]
On the contrary, the observation of the production cytokines tells us a lot about the underlying type of immune response that is occurring as production of particular cytokine are associated with Type I immune response, Type II immune responses, etc., and typically reflect not only the specific T cell types that have been activated, but also which types of antigen presenting cells are involved as well.  This is key biology, not just some random phenomenology, as the type of immune response often determines whether the response to an infection will be successful as some pathogens require stronger T effector cell involvement, and others require stronger antibody based responses, and there are particular patterns of cytokine production associate with each.

Let's keep the GO useful for immunologists.

**Comment by BarbaraCzub** [comment]
Thank you @ukemi @pgaudet and @addiehl for all your comments. The aim of Shirin's project is to capture the roles of various microRNAs in regulating interleukins, and so these terms provide contextual information, which can provide a valuable indication of what a given microRNA is doing. I'll add the term, as per Shirin's request, and as supported by @addiehl 's comment. Thanks!

**Comment by pgaudet** [comment]
Hi @addiehl 
It would be great if these terms were somehow connected to immune response (see screenshot below). Could they be part_of 'GO:0006955 immune response'? 

<img width="240" alt="image" src="https://user-images.githubusercontent.com/4782928/70422546-33702880-1a6c-11ea-9b17-b71f1057b1a7.png">

Thanks, Pascale

**Comment by pgaudet** [comment]
How about 'immune response-regulating signaling pathway'? would that be a good parent ?

**Comment by BarbaraCzub** [comment]
There is a GO term for 'GO:0002367 cytokine production involved in immune response'. But the 'GO:0006954 inflammatory response' should probably act as another parent. And 'inflammatory response' should probably also become a part of 'GO:0045087 innate immune response'.

The immune and inflammatory response are not currently related:
![inf and immune tree](https://user-images.githubusercontent.com/16541096/70512674-18bbb380-1b41-11ea-938c-c722629e04cb.png)

And here is how the cytokines fit (or do not fit) into this scheme (image can be enlarged by clicking on it):
![cytokines and inf and immune tree](https://user-images.githubusercontent.com/16541096/70512702-2cffb080-1b41-11ea-8e08-4a112858e0dc.png)

**Comment by BarbaraCzub** [comment]
Hi @pgaudet, @RLovering @ShirinSaverimuttu and I had a discussion about these potential ontology improvements today, and since it would be a very large project to establish which of the cytokines/interleukins are involved in immune responses, which ones in inflammatory responses, and which ones in both, we are not able to address this issue as a part of our ongoing work. I'll close this issue now, as this would be better addressed as a part of a dedicated project.

**Comment by addiehl** [comment]
I totally agree with this assessment, and of course there's the fundamental issue of the definition of cytokine, which is basically any protein secreted by one cell that has an effect on the activity of another cell.  This includes things like growth factors and protein/peptide hormones, as well as local and systemic effects.  Some cytokines in the immune system are involved in immune developmental processes that are not immune or inflammatory responses, but may also have roles in other situations in immune or inflammatory responses.

The key thing for me is that these terms should be annotated to so that they appear in term enrichment results and experimental researchers assess the relevance of the results to the specific system under study.

**Comment by addiehl** [comment]
As far as the relation between inflammatory response and immune response, when doing the GO immunology revision, we attempted to isolate processes specifically related to the definition of inflammatory response "The immediate defensive reaction (by vertebrate tissue) to infection or injury caused by chemical or physical agents. The process is characterized by local vasodilation, extravasation of plasma into intercellular spaces and accumulation of white blood cells and macrophages."  Of course many immune responses result in inflammatory responses, but the response to simple injury, like a skin cut, is also an inflammatory response, but may or may not be an immune response.

Based on the definition of 'immune response', "Any immune system process that functions in the calibrated response of an organism to a potential internal or invasive threat," while the general definition of inflammatory response seems to be a "calibrated response of an organism to a potential internal or invasive threat," we were not sure if all parts of inflammatory responses and all types of inflammatory responses were in fact immune system processes.  Thus we chose not to make 'inflammatory response' a subtype of 'immune response'.

I would be happy to discuss this distinction further in this or another forum.

**Comment by BarbaraCzub** [comment]
> As far as the relation between inflammatory response and immune response, when doing the GO immunology revision, we attempted to isolate processes specifically related to the definition of inflammatory response "The immediate defensive reaction (by vertebrate tissue) to infection or injury caused by chemical or physical agents. The process is characterized by local vasodilation, extravasation of plasma into intercellular spaces and accumulation of white blood cells and macrophages." Of course many immune responses result in inflammatory responses, but the response to simple injury, like a skin cut, is also an inflammatory response, but may or may not be an immune response.
> 
> Based on the definition of 'immune response', "Any immune system process that functions in the calibrated response of an organism to a potential internal or invasive threat," while the general definition of inflammatory response seems to be a "calibrated response of an organism to a potential internal or invasive threat," we were not sure if all parts of inflammatory responses and all types of inflammatory responses were in fact immune system processes. Thus we chose not to make 'inflammatory response' a subtype of 'immune response'.
> 
> I would be happy to discuss this distinction further in this or another forum.

This is exactly what Ruth said when we had our discussion today! Thank you, @addiehl, for these helpful comments.

**Comment by RLovering** [comment]
Hi Both
I have just been looking at this part of the ontology with a clinician with an interest in the innate inflammatory response, in particular IL6.
My arguement, like Alex's has always been that inflammation occurs without an infection or toxins, however the problem is that the innate immune cells are involved in inflammation, and that the body does have to detect a change in the tissue or cells that leads to an inflammatory response. So we were discussing if an immune response always leads to inflammation, the exception might be vaccination, but this is often associated with aching joints or raised temperate.

So I am now sitting on the fence on this one, but if these were to be linked in any way I would make inflammation the part_of parent to the immune response, because not every occurance of inflammation is an immune response, whereas I think an immune response leads to some sort of inflammation.

Finally I am very concerned about the current ontology. Why isn't immune response a child of  GO:0098542 defense response to other organism?
![Screenshot 2019-12-12 at 15 09 00](https://user-images.githubusercontent.com/12080010/70723772-5c551f80-1cf1-11ea-9808-3e6af1e980ab.png)

**Comment by addiehl** [comment]
'immune response' is not a child of 'defense response to other organism' because regulatory immune responses are not defense responses and autoimmune responses are not defense responses.

Note that not all autoimmune responses are 'abnormal'.  Some occur as regulatory immune responses to suppress other autoimmune responses.

**Comment by RLovering** [comment]
Good point but there should be an adaptive immune response child term that has the parent defense response to other organism. You can't expect curators to realise that in order to be consistant in the annotation of innate immune response genes and adaptive immune response genes they have to add the additional defense response annotation
I have just used QuickGO to look at the reviewed human proteins associated with adaptive immune response and identified 600 proteins associated with this term (obviously lots of IG and TCR genes included). Of these 600 proteins:

Also associated with defense response
150 proteins (not Ig or TCR or HLA)
74 IG
13 TCR
16 HLA

Only associated with adaptive immune response (ie NOT associated with defense response)
140 proteins (not Ig or TCR or HLA)
87 IG
113 TCR
4 HLA

I think a GO term is needed along the lines of  GO:0090720 primary adaptive immune response, and  GO:0140367 antibacterial innate immune response etc. ie 
antibacterial adaptive immune response
Then there could be the parent:
antibacterial immune response 
which would also have the child term:  GO:0032496 response to lipopolysaccharide
Perhaps we could identify all proteins associate with both 
 GO:0032496 response to lipopolysaccharide and either adaptive immune response OR innate immune response and annotate them to the relevant antibacterial immune response term.

 GO:0042742 defense response to bacterium
>is_a child NEW GO term: antibacterial immune response
>>is_a child GO:0140367 antibacterial innate immune response
>>is_a child GO:0032496 response to lipopolysaccharide
>>is_a child NEW GO term: antibacterial adaptive immune response

And repeat as above for virus and fungi.

I can send you the analysis mentioned above if you would like me to

Ruth

**Comment by addiehl** [comment]
It would be great to see the analysis (email it directly or attach it here).

I can see the value in terms like 'antibacterial adaptive immune response' as a way of have a child of both adaptive immune response and defense response, particularly as a way of simplifying the annotation process.  I think that one reason I have not previously seen the need for this type of defined class is that any gene product that that is annotated to adaptive immune response could in theory be used for any subtype of adaptive immune response to particular organisms, that the whole point of being adaptive is that in theory antigens from any source can be targeted, and a memory enhanced immune response will be produced.

Of course there are things like NK T cells and some classes of gamma-delta T cells that have TCRs preselected for antigens of bacterial origin because of the use of particular V gene segments.  But arguably, the immune response driven by these cells are really innate rather than adaptive, or perhaps innate with processes typically associated with adaptive immune memory occurring in the T cells.

TL;DR. Terms like 'antibacterial adaptive immune response' are okay with me.

**Comment by pgaudet** [comment]
This ticket is closed - should we open ?

**Comment by addiehl** [comment]
Should probably open a new ticket based on recent posts.

**Comment by RLovering** [comment]
Thanks Alex

I hope the attached makes sense.

Note that I filtered the defence response annotations using the human protein IDs from the adaptive immune response annotations, so this isn’t a list of all the proteins associated with this term

Best

Ruth

**Comment by addiehl** [comment]
Hi Ruth, sorry to bother, but I'm not seeing an attachment.  Could be my unfamiliarity with the Github interface.  Can you advise?

**Comment by RLovering** [comment]
[DefenceResponsevAdaptiveResponse.xlsx](https://github.com/geneontology/go-ontology/files/3957552/DefenceResponsevAdaptiveResponse.xlsx)

**Comment by RLovering** [comment]
Sorry I thought I had attached it to the email I sent, I have added it to the github ticket, hopefully

**Comment by addiehl** [comment]
Thanks

**Comment by ValWood** [comment]
Do we really need to encode the organism in the immune response terms?

In reviewing the ontology and the existing annotations, most of the annotation inconsistencies  are due to some curators annotating to terms under "innate immune response" and some to "defence response to fungus".

It would require A LOT of duplication to create 

to cover the existing
response to fungus
response to protazoan
response to bacterium
response to insect.

The "other organism" can be captured by the taxon?

Curators are  finding the high-level "response to x " terms, but not the terms under "innate immune response"

So for example for plants, the pathway activated is often

GO:0009626    plant-type hypersensitive response
we would not really want to have

 plant-type hypersensitive response in  response to fungus
plant-type hypersensitive response in  response to virus
plant-type hypersensitive response in  response to oomycete

It  would be much better to concentrate on pathway and mechanism, and take species out of the equation (or capture species in and other way)

One alternative would be to allow "in response to fungus/bacteria" as an extension, in much the same way as we can have "phases" in extensions now.

**Comment by addiehl** [comment]
Clearly, the original approach avoided the duplication of hierarchies Val warns of, but on the other hand, Ruth and her group do annotation in this area and point out the advantages of additional pre-composed terms.  I don't have a strong opinion either way, though I continue to see the value of pre-composed terms as seemingly a faster approach to annotation than complex GO-CAM assemblages.  But I don't currently do annotation so perhaps my sense of the relative difficulty of building up GO-CAM representations from simpler classes vs using "traditional" precomposed terms is not supported by practice.

**Comment by ValWood** [comment]
All GO curators should be familiar with annotation extensions and their use - we have been using them since before 2014, so this shouldn't be an issue.

One thing for sure is that the current situation results in suboptimal and inconsistent annotation.

Many people are annotating only to the "response to fungus" terms  which do not have children, rather than biologically  informative terms. I recoreded and example here:

https://github.com/geneontology/go-ontology/issues/18072

By follwing the "response to fungus" branch my collaborator annotated to
GO:0038187 | pattern recognition receptor activity | IMP | B | part_of cell surface pattern recognition receptor signaling pathway , part_of response to fungus


GO:1900150 | regulation of defense response to fungus
(which has no descendants)
For the same annotation, I did

GO:0038187 | pattern recognition receptor activity | IMP | B | part_of cell surface pattern recognition receptor signaling pathway , part_of response to fungus

Extensions should not be a problem as it is easy to configure the tools to suggest the appropriate extension types for specific terms.

However, my preferred solution would be to get rid of the "response to species" and concentrate on the biological mechanism. I can see why this might be a step too far though...but we definitely need to do something.

**Comment by addiehl** [comment]
I always like to think of the value of GO terms and their annotations in how they show up in term enrichment results (or ideally would do so if annotation were more complete).  The challenge with annotation extensions is that term enrichment algorithms do not consider or show these things in results, and thus they have no value to 98% of the consumers of GO.  Ideally we need better tools for displaying and analyzing the full range of information we collect as part of the GO annotation process, and I am not arguing we should not make complex GO annotations.  But if we remove terms such as the response to species ones, we remove the ability to collect valuable contextual information in the annotation process contained in these term names and definition that will appear in (and has always appeared in) term enrichment results and is surely of value to our users.

**Comment by ValWood** [comment]
But "response to fungus" is not as informative in an enrichment than a  biological process or pathway. Consistency improves enrichments and currently, the useful terms are really hidden by these "response to organism"  branches. I would much rather have real processes and signalling pathways in my enrichment, and then figure out if it was relevant to my pathogen of interest  (which  presumably provided the genes in my experimental input ? so it isn't really telling me anything new?).

Plus I would now need to request "response to fungal specific terms for all of the processes I have annotated, linked to the "response to fungus" terms and move my annotations down to them, even if the editors agree to do this.

Extensions provide a useful way to capture this information which is slightly orthogonal, and hopefully, by the time there are enough annotations to be useful  the tools will have caught up 
and allow enrichment over extensions.

**Comment by RLovering** [comment]
Hi Val 
I made this suggestion for 2 reasons:
1. currently adaptive immune response is not a child of defense response and curators do not realise this and therefore there is inconsistent annotation of proteins involved in adaptive immunity:
approx 250 human proteins associated with adaptive immune response AND defense response
150 proteins (not Ig or TCR or HLA)
74 Immunoglobulin (ig)
13 T-cell receptor (TCR)
16 HLA

approx 320 human proteins ONLY associated with adaptive immune response (ie NOT associated with defense response)
140 proteins (not Ig or TCR or HLA)
87 IG
113 TCR
4 HLA

So I thought we need a term that will enable a curator to annotate to both adaptive immune response AND defense response without having to create 2 annotations.

2. The innate immune response term has child terms (see figure below), which look like they were created this year, or maybe just were edited recently, and I thought it made sense to have the equivalent terms in the adaptive immune response. 

Although now you mention it you are probably right that, as the adaptive immune response is adaptive, perhaps none of the proteins are organism/antigen specific, which they are in the innate immune response. I just thought that some of the regulators of the adaptive immune response might be organism/antigen specific. 

Maybe we could just have adaptive immune response involved in defense response?

![Screenshot 2019-12-13 at 17 35 56](https://user-images.githubusercontent.com/12080010/70819889-3c942900-1dcf-11ea-898c-531c978aa63a.png)

**Comment by ValWood** [comment]
Yes, this is *exactly* the same issue I had  with the fungal/innate immune response annotations. Everything was annotated in one branch of the other, but not both  (in fact there were  even more options but we now got rid of some).
 "fungal innate immune response" existed.  It has 13 experimental annotation.

I have been using the specific pathways like
"plant-type hypersensitive response"  and "GO:0002221  pattern recognition receptor signaling pathway"which are the 2 main plant responses to fungi . Possibly to other organisms too, I don't know. 
They don't, and probably can't have "antifungal innate immune response parentage".
But this isn't as useful as knowing the biological pathway IMHO.

I don't know how useful the terms are  if these do not actually represent different processes.  It seems a lot of work to duplicate (and outside the scope of GO) 
So the question is: How is the antiviral innate immune response different from the antifungal innate immune response? I don't know the answer becasue I haven't annotated any viruses.
..and maybe the answer is different for different host species?

**Comment by cmungall** [comment]
@addiehl :
> Should probably open a new ticket based on recent posts.

I agree, the conversation is outside the scope of the original ticket.

**Comment by RLovering** [comment]
I have opened a new ticket and will copy some of this discussion to that ticket
